### PR TITLE
Bind QuickMenuViewModel to QuickMenuView

### PR DESCRIPTION
**Pull request by codeimpl**
This patch binds view model to view.

Refs: #973

### CODE REVIEW

File: src/org/triple/banana/quick_menu/QuickMenuControllerImpl.java [hunk 1]
```diff
@@ -34,6 +34,7 @@ private void loadButtons() {
     @Override
     public void onClickQuickMenuButton(@IdRes int buttonId) {
         mQuickMenuActionProvider.getAction(buttonId).doAction();
+        dismiss();
```
Comment (codeimpl) [inline src/org/triple/banana/quick_menu/QuickMenuControllerImpl.java:37]: Current, we will close dialog for all menu.

File: src/org/triple/banana/quick_menu/QuickMenuButton.java [hunk 2]
```diff
@@ -52,8 +52,8 @@ private void initialize(Context context) {
         mTextView.setLayoutParams(textViewParams);
     }
 
-    public void setLabel(String str) {
-        mTextView.setText(str);
+    public void setLabel(int resId) {
```
Comment (jdragonbae) [inline src/org/triple/banana/quick_menu/QuickMenuButton.java:55]: With annotation?
`public void setLabel(@IdRes int resId)`

File: src/org/triple/banana/quick_menu/QuickMenuDialog.java [hunk 3]
```diff
@@ -51,15 +55,17 @@ public void onContentChanged() {
...
-        addQuickMenuButtonToGridLayout();
     }
 
-    private void addQuickMenuButtonToGridLayout() {
-        // FIXME(#973): Bind button with QuickMenuViewModelData.
-        for (int i = 0; i < 16; i++) {
-            QuickMenuButton button = new QuickMenuButton(getContext());
-            button.setLabel(Integer.toString(i));
-            button.setImageResource(R.drawable.btn_done);
+    private void updateButtons(@NonNull List<ButtonInfo> buttons, View.OnClickListener listener) {
```
Comment (jdragonbae) [inline src/org/triple/banana/quick_menu/QuickMenuDialog.java:60]: I think View.OnClickListener listener would be @NonNull. WDYT?

File: src/org/triple/banana/quick_menu/QuickMenuButton.java (hunk 2 above)
Comment (codeimpl) [reply src/org/triple/banana/quick_menu/QuickMenuButton.java]: sure thanks!

File: src/org/triple/banana/quick_menu/QuickMenuDialog.java (hunk 3 above)
Comment (codeimpl) [reply src/org/triple/banana/quick_menu/QuickMenuDialog.java]: done! :)

File: src/org/triple/banana/quick_menu/QuickMenuButton.java (hunk 2 above)
Comment (codeimpl) [reply src/org/triple/banana/quick_menu/QuickMenuButton.java]: I updated as @StringRes :)

**Review by romandev — APPROVED** [review]
lgtm

**Pull request merged**